.circleci/config.yml

```diff
@@ -1,47 +1,26 @@
 version: 2.1
 
-parameters:
-  release-name:
-    type: string
-    default: "runner-init"
-
 orbs:
   slack: circleci/slack@4.13.3
 
-x-data:
-  linux_machine_image: &linux_machine_image ubuntu-2204:2024.02.7
-  go_image: &go_image cimg/go:1.22
-
 executors:
   go:
     docker:
-      - image: *go_image
+      - image: cimg/go:1.22
     resource_class: circleci-runner/rum-large
+  linux_machine:
+    machine:
+      image: ubuntu-2204:2024.02.7
+      resource_class: large
 
 workflows:
   main-workflow:
     jobs:
       - lint
       - build
-      - build-and-publish-images:
-          name: build-and-publish-image-amd64
-          context: org-global
+      - images:
+          context: [ org-global ]
           requires: [ lint, build ]
```

Comment:
These images are really small, and we use xbuild to build multi-arch images, so let's just do this all within a single job/machine.